storage.js

```diff
@@ -0,0 +1,32 @@
+function saveDataForm() {
+  const inputs = document.querySelectorAll('input');
+
+  const dataForm = {};
+
+  inputs.forEach((input) => {
+    dataForm[input.id] = input.value;
+  });
+
+  localStorage.setItem('dataForm', JSON.stringify(dataForm));
+}
+
+function loadFormData() {
+  const savedData = localStorage.getItem('dataForm');
+
+  if (savedData) {
+    const dataForm = JSON.parse(savedData);
+
+    Object.keys(dataForm).forEach((field) => {
+      if (Object.prototype.hasOwnProperty.call(dataForm, field)) {
+        document.getElementById(field).value = dataForm[field];
+      }
+    });
+  }
+}
+
+const inputs = document.querySelectorAll('input');
```

Comment:
- [ ] you did a great job however that an issue with the message (textarea). When the user add a name and email the reload the page, the name and email are still there while the message is not displayed. Kindly fix that issue.